### DISCUSSION TITLE
Max height for claims!

### DISCUSSION THREAD

**Discussion by josedabela** [GriefPrevention/GriefPrevention#2280] [Ideas]
Make it so we can change in the configs the max height of the claim and not be infinite!..

**Comment by QarthO** [comment]
Are you thinking maxmium height as in size of the claim, or maximum y level for the claim?

**Reply by josedabela** [reply]
The Y level .. there should be a option in the configs that we could change that. Like the claim will only be protected 30 block up for example.

**Comment by josedabela** [comment]
The Y level .. there should be a option in the configs that we could change that. Like the claim will only be protected 30 block up for example.
